cursorless-talon/src/marks/literal_mark.py

```diff
@@ -0,0 +1,28 @@
+from talon import Module
+
+from .mark_types import LiteralMark
+
+mod = Module()
+
+
+# FIXME: Replace literal *heh* with a Talon list before we make this public
+@mod.capture(rule=("literal <user.text>"))
+def cursorless_literal_mark(m) -> LiteralMark:
+    return {
+        "type": "literal",
+        "modifier": {
+            "type": "preferredScope",
+            "scopeType": {
+                "type": "customRegex",
+                "regex": construct_fuzzy_regex(m.text),
+                "flags": "gui",
+            },
+        },
+    }
+
+
+def construct_fuzzy_regex(text: str) -> str:
+    parts = text.split(" ")
+    # Between each word there can be nothing(camelCase) or a non character symbol.
+    # Escape characters. eg: \t\r\n, are also acceptable.
```

Comment:
I had a bit of trouble parsing `nothing(camelCase)`. Does it mean
> there can be nothing (e.g., when searching for a camelCase identifier)

Is this version of the comment equally accurate?

```suggestion
    # Each word can optionally be separated by a sequence of non-letter characters (as well as \t, \r, or \n).
```

Reply:
More or less. Each word can be separated by nothing. `helloWorld` are an example of two words separated by nothing.